frontend/packages/kubevirt-plugin/src/components/form/form-row.tsx

```diff
@@ -39,30 +40,31 @@ export const FormRow: React.FC<FormRowProps> = ({
         type === ValidationErrorType.Info || type === ValidationErrorType.Warn ? message : undefined
       }
       className={className}
-    >
-      {help && (
-        <span className="kubevirt-form-row__icon-status-container">
-          <PopoverStatus
-            title={`${fieldId} help`}
-            hideHeader
+      labelIcon={
+        help && (
+          <Popover
+            position={PopoverPosition.right}
+            aria-label={`${fieldId} help`}
             onHide={onHide}
             onShow={onShow}
-            statusBody={
-              <StatusIconAndText
-                title={`${fieldId} help`}
-                iconOnly
-                icon={
-                  <HelpIcon
-                    className={classnames({ 'kubevirt-form-row__help-icon--hidden': !isActive })}
-                  />
-                }
-              />
-            }
+            bodyContent={help}
           >
-            {help}
-          </PopoverStatus>
-        </span>
-      )}
+            <button
+              type="button"
+              onClick={(e) => e.preventDefault()}
+              className="pf-c-form__group-label-help"
+            >
+              <HelpIcon
+                noVerticalAlign
+                className={classnames({
+                  'kubevirt-form-row__help-icon--hidden': !isActive,
```

Comment:
can we remove this class? I'm not sure what it's purpose is but it's adding a color override.

Reply:
I removed the class and the logic that was used to apply it. I'm not sure why it was added originally, but I didn't see it being used anywhere else, so it doesn't appear to be any type of convention used for consistency.